### PR TITLE
script to run mno analysis

### PR DESCRIPTION
**Pull request by DanielMwendwa**

### CODE REVIEW

File: mno_analysis_tools/run-mno-analysis.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+if [[ $# -ne 9 ]]; then
+    echo "Usage: ./run_pipeline.sh"
```
Comment (IsaackMwenda) [inline mno_analysis_tools/run-mno-analysis.sh:6]:
```suggestion
    echo "Usage: ./run-mno-analysis.sh"
```

File: mno_analysis_tools/run-mno-analysis.sh [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    echo " [--profile-memory <profile-output-path>]"
+    echo " <domain> <token> <raw_messages_file_path>"
+    echo " <target_operator> <target_message_direction>"
+    echo " <start_date> <end_date> <time_frame> <optional_time_frame>"
+    echo " <output_dir>"
+    echo "Runs the pipeline end-to-end (Fetch Raw Messages, compute window of downtime,
```
Comment (IsaackMwenda) [inline mno_analysis_tools/run-mno-analysis.sh:12]:
```suggestion
    echo "Runs the Mno Analysis end-to-end (Fetch Raw Messages, compute window of downtime,
```

File: mno_analysis_tools/run-mno-analysis.sh (hunk 1 above)
Comment (DanielMwendwa) [reply mno_analysis_tools/run-mno-analysis.sh]: Updated thanks

File: mno_analysis_tools/run-mno-analysis.sh (hunk 2 above)
Comment (DanielMwendwa) [reply mno_analysis_tools/run-mno-analysis.sh]: Updated thanks

**Pull request merged**